src/content/docs/quick-tips/docker-image.mdx

```diff
@@ -54,7 +53,7 @@ services:
 * `4001:4001/tcp` binds local port 4001 to the same container port. Port 4001 is the default for the UI and is necessary to expose the app. 
 * `8888:8888/udp` binds the localhost port 8888 (default for OSC input into the app) to the same in the container, so that it can receive data from the host machine targeted to the loopback interface.
 * `8888:8888/udp` binds the localhost port 9999 (default for OSC output from the app) to the same in the container, this allows Ontime to send data out of the container.
-* `"./ontime-db:/external/projects/"`, `"./ontime-styles:/external/styles/"` and `"./ontime-external:/external/external/"` expose the internal container directories.
```

Comment:
i think `external/projects/` is correct....

Reply:
ok reverted the changes, lets check this with users